### PR TITLE
User can Create event

### PR DESCRIPTION
**Pull request by dernathul**
# [PT Story](https://www.pivotaltracker.com/story/show/172266208)
```
As a User,
In order to create an event
i need to have a form to fill up and send
```
## Changes proposed in this pull request:
* Form for creation of event added
* Test for happy and sad paths
* Some minor styling
* Communication with the back end for event creation

## What I have learned working on this feature:
* Read legacy code and refactor
* Create instances to send different messages
* Work with semantic UI for styling
* Using google to find solution

## Screenshots (Only if client side story)

Screenshot 2020-04-20 at 14 08 59
Screenshot 2020-04-20 at 14 09 06
Screenshot 2020-04-20 at 14 09 21
Screenshot 2020-04-20 at 14 09 36
Screenshot 2020-04-20 at 14 09 54

### CODE REVIEW

File: src/components/EventsList.jsx [hunk 1]
```diff
@@ -1,34 +1,37 @@
...
 
     fetchEvents();
   }, []);
 
+ 
+  
+
   return (
     <>
+
```
Comment (hmvitous) [inline src/components/EventsList.jsx]: Check formatting on this file

File: src/App.jsx [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
-          <EventsList />
-        </div>
-      </>
-    );
-  }
-}
+import { Button } from "semantic-ui-react";
+import EventCreate from "./components/EventCreate";
+
+
```
Comment (hmvitous) [inline src/App.jsx]: Take away the extra line

File: .env [hunk 3]
```diff
@@ -0,0 +1 @@
+PORT=3001
```
Comment (hmvitous) [inline .env:1]: I think this file should be in .gitignore

**Review by hmvitous — CHANGES_REQUESTED** [review]
just a couple formatting things to address

File: .env (hunk 3 above)
Comment (dernathul) [reply .env:1]: well, when it was there i was having a lot of problems
this is why i took it out...mr google told me it was a good solution and i think it worked

**Comment by hmvitous** [comment]
Change the title and it should be fine

**Pull request merged**